ui/pages/remote-mode/remote.types.ts

```diff
@@ -1,8 +1,10 @@
 // Note: just some initial types (will likely be refactored to pull from contract metadata
 
+import { AssetType } from '@metamask/bridge-controller';
 import { Delegation } from '../../../shared/lib/delegation';
 
 export enum TokenSymbol {
+  SEPOLIA_ETH = 'SepoliaETH',
```

Comment:
OK for now, but we'll need to think about we handle testnet tokens going forward (as we won't want them showing up when in release builds).